pkg/cloud/services/compute/instance.go

```diff
@@ -94,7 +94,7 @@ func (s *Service) normalizePortTarget(port *infrav1.PortOpts, openStackCluster *
 				if err != nil {
 					// Multiple matches might be ok later when we restrict matches to a single network
 					if errors.Is(err, networking.ErrMultipleMatches) {
-						s.scope.Logger().V(4).Info("Can't infer network from subnet %d: %s", i, err)
+						s.scope.Logger().V(4).Info("Couldn't infer network from subnet","IPIndex", i, "err", err)
```

Comment:
I would probably change IPIndex to SubnetIndex: FixedIPs is a weird naming thing from OpenStack which really means subnets.

However, both this and the above are very much debug level logs, so anybody interested is likely also looking at the code.